coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelToAvroSchemaConverter.java

```diff
@@ -272,7 +272,14 @@ public RelNode visit(LogicalUnion logicalUnion) {
       RelNode relNode = super.visit(logicalUnion);
       Schema inputSchema1 = schemaMap.get(logicalUnion.getInput(0));
       Schema inputSchema2 = schemaMap.get(logicalUnion.getInput(1));
-
+      /* forceLowercase is only used for the union of schemas so column name will match
```

Comment:
I am not following, looks like this PR converts the final schema to lowercase if `forceLowercase =true` for all the schemas, why do we say it's only used for union of schemas?

Reply:
Yeah the implementation makes sense, I was just concerned about the comment, I meant `forceLowercase` is not just used for `union`.

Reply:
Sounds good, I will change this comment. By the way you mentioned `Please make sure the affected views can be queried on gateway with this PR.`, is there a document for that?